personio-personnel-data-api-oa3.yaml

```diff
@@ -2216,6 +2216,128 @@ paths:
                           "The file must be a file of type: pdf, docx, doc, jpg, zip, png, txt, jpeg, odt, xlsx, rtf, htm, xls, p7s, pptx, pages, rar, ppt, gif, tif, html, msg, asc, tiff, wps, bmp, 7z, csv, ics, vcf, dotx, numbers, eps, gz, ods, otf, odp, odg, rtfd, txz, webarchive, xml, xps, md, fax."
                         ]
 
+  /company/custom-reports/reports:
+    get:
+      tags:
+        - Custom Reports
+      parameters:
+        - name: X-Company-Id
+          in: header
+          schema:
+            type: integer
+          required: true
+        - name: report_ids
+          in: query
+          schema:
+            type: array
+            items:
+              type: string
+          required: false
+        - name: status
+          in: query
+          schema:
+            type: string
+          example: "up_to_date"
+          required: false
+      summary: List of reports
+      operationId: listReports
+      responses:
+        200:
+          description: Ok
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicListReportsResponse"
+        500:
+          description: Internal Server Error
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicAPIException"
+
+  /company/custom-reports/reports/{reportId}:
+    get:
+      tags:
+        - Custom Reports
+      parameters:
+        - name: X-Company-Id
+          in: header
+          schema:
+            type: integer
+          required: true
+        - name: reportId
+          in: path
+          schema:
+            type: string
+          required: true
+        - name: locale
+          in: query
+          description: locale used to translate localized fields
+          schema:
+            type: string
+          required: false
+      summary: Get report with items
+      operationId: listReportItems
+      responses:
+        200:
+          description: Ok
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicReportResponse"
+        500:
+          description: Internal Server Error
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PublicAPIException"
+
+  /company/custom-reports/columns:
+    get:
+      tags:
+        - Custom Reports
+      parameters:
+        - name: X-Company-Id
+          in: header
+          schema:
+            type: integer
+          example: 1
+          required: true
+        - name: columns
+          in: query
+          schema:
+            type: array
+            items:
+              type: string
+          example: first_name,last_name
+          required: false
+        - name: locale
+          in: query
+          schema:
+            type: string
+          example: "en"
+          required: false
+        - name: report_id
```

Comment:
[Suggestion]: Here we are using `report_id` again. Let's make it consistent by using `report_id` everywhere (referring to the previous GET request).